docs/pages/versions/unversioned/sdk/filesystem.md

```diff
@@ -522,3 +563,21 @@ FileSystem.getTotalDiskCapacityAsync().then(totalDiskCapacity => {
 Returns a Promise that resolves to a number that specifies the total internal disk storage capacity in bytes, or JavaScript's [`MAX_SAFE_INTEGER`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Number/MAX_SAFE_INTEGER) if the capacity is greater than 2<sup>53</sup> - 1 bytes.
 
 #
+
+## Supported URI schemes
+
```

Comment:
We might want to add some caption to the graph, something like "In this table, you can see what file URI's could be used per method.". That should bring people not 100% familiar with Expo up to speed before the information bombardment 😄

Reply:
> We might want to add some caption to the **graph**

I think you mean _"Table"_ 😉  That's really good idea! 😃  I'll add some introduction text.

Reply:
haha yeah, my bad!

Reply:
Ok, I've added a brief description to the table.